src/main/java/it/gov/pagopa/payment/service/payment/common/CommonPreAuthServiceImpl.java

```diff
@@ -91,43 +94,37 @@ public AuthPaymentDTO previewPayment(TransactionInProgress trx, String channel)
   protected void checkPreAuth(String trxCode, String userId, TransactionInProgress trx) {
     String walletStatus = walletConnector.getWallet(trx.getInitiativeId(), userId).getStatus();
     if (PaymentConstants.WALLET_STATUS_SUSPENDED.equals(walletStatus)){
-      throw new ClientExceptionWithBody(
-              HttpStatus.FORBIDDEN,
+      throw new UserSuspendedException(
```

Comment:
some Exception, like this, could have a default code that is supposed to be always the same: we could thus expose a constructor without CODE, using such default code in the constructor